var/spack/repos/builtin/packages/py-statsmodels/package.py

```diff
@@ -27,7 +29,7 @@ class PyStatsmodels(PythonPackage):
     # according to https://www.statsmodels.org/dev/install.html earlier versions
     # might work.
     depends_on('py-setuptools@0.6c5:', type='build')
-    depends_on('py-cython@0.29:', type='build')
+    depends_on('py-cython@0.29.26:', type='build')
```

Comment:
Is this minimum version requirement also true for older versions?

Reply:
I have no idea what minimum versions of py-cython were for older versions. The only thing I can find from the statsmodels website is 0.29.26. Maybe I should use 
depends_on('py-cython@0.29.26:', type='build', when='@0.13.0:')
to be safe?

Reply:
I never look at the website, I only look at the tarball.

Reply:
I took a quick look at the tarballs for statsmodels 0.13.0-0.13.2, requirements-dev.txt always has 
cython>=0.29

So maybe keeping minimum version requirement of cython untouched is fine. Maybe just merge the PR as it is?

Reply:
I also never look at requirements.txt since it isn't used by the build system. The only things that matter are pyproject.toml, setup.cfg, and setup.py. See https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#build-backend